.github/CODEOWNERS

```diff
@@ -808,6 +808,9 @@ sdk/ai/ai-projects @ganeshyb @glharper @dargilco @jhakulin @ZachhK
 # PRLabel: %Mgmt
 /sdk/postgresql/arm-postgresql/ @qiaozha @MaryGao
 
+# PRLabel: %Mgmt
+/sdk/purestorageblock/arm-purestorageblock/ @qiaozha @MaryGao
```

Comment:
We will keep the code owner in our side for now, but eventually change the code owner as service team.